### ISSUE TITLE
Publish on NPM, or not yet

### ISSUE THREAD

**Issue by fpoli** [viperproject/vs-verification-toolbox#3]
At this point, we could publish this project on NPM, but that would add a bit of overhead (bump version numbers, publish every now and then).

Since this project is currently just used in our group, I think that it might be easier if dependant packages point to `vs-verification-toolbox` using the git address and commit or branch name (e.g. see [this](https://stackoverflow.com/questions/14187956/npm-install-from-git-in-a-specific-version)).

As soon as a dependant package needs to be published to NPM, or if some non-Viper project wants to use `vs-verification-toolbox`, then we could start actually using NPM.

@aterga @Felalolf @ArquintL What do you think? For example, do you plan/need to publish the Viper/Gobra IDE package on NPM?

**Comment by ArquintL** [comment]
This approach works for us. 
I tried to install the toolbox a couple of days ago and had problems to get TS to find the module (I've used `npm install --save-dev viperproject/vs-verification-toolbox`). Silas also had similar problems but he's currently looking into this with Julian. Long story short it's important to me that that the toolbox works like any other npm module but I don't care whether you install it by specifying a dependency to this repo or to the npm registry.
Regarding Gobra IDE & NPM: we currently do not intend to do so unless this is somehow a requirement (e.g. for a release in the VSCode store)
